spec/datadog/core/telemetry/v1/telemetry_request_spec.rb

```diff
@@ -164,4 +164,52 @@
       end
     end
   end
+
+  describe '#to_h' do
+    subject(:to_h) { telemetry_request.to_h }
+
+    let(:api_version) { 'v1' }
+    let(:application) do
+      Datadog::Core::Telemetry::V1::Application.new(language_name: 'ruby', language_version: '3.0',
+                                                    service_name: 'myapp', tracer_version: '1.0')
+    end
+    let(:debug) { false }
+    let(:host) { Datadog::Core::Telemetry::V1::Host.new(container_id: 'd39b145254d1f9c337fdd2be132f6') }
+    let(:payload) do
+      Datadog::Core::Telemetry::V1::AppStarted.new(
+        integrations: [Datadog::Core::Telemetry::V1::Integration.new(
+          name: 'pg', enabled: true
+        )]
+      )
+    end
+    let(:request_type) { 'app-started' }
+    let(:runtime_id) { '20338dfd-f700-0d470f054ae8' }
+    let(:seq_id) { 42 }
+    let(:session_id) { '20338dfd-f700-0d470f054ae8' }
```

Comment:
Not that it matters a lot for this test, but these are documented as V4 UUIDs in the code, and these two examples are not V4 UUIDs which I find a bit confusing

My suggestion is, if we're using dummy data, we could using something like `dummy_runtime_id` rather than something that _looks like_ real data, but isn't. 😄